src/twisted/internet/defer.py

```diff
@@ -1898,6 +1898,7 @@ def __init__(self, value: object) -> None:
         self.value = value
 
 
+@deprecated(Version("Twisted", "NEXT", 0, 0))
```

Comment:
Maybe also hint about the replacement.

```suggestion
@deprecated(Version("Twisted", "NEXT", 0, 0), replacement="standard return statement")
```

Reply:
Good idea.